Header      hdr
char       	is_disabled
char       	is_enabled
char       	is_paused
char       	is_fault
